examples/blinky/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 cmake_minimum_required(VERSION 3.16)
 set(CMAKE_TOOLCHAIN_FILE ${CMAKE_CURRENT_SOURCE_DIR}/../../cmake/stm32_gcc.cmake)
 
+set(STM32_CUBE_F4_PATH "C:\\GIT\\STM32CubeF4")
```

Comment:
Use the `stm32_fetch_cube(F4)` function, in the if stm32_fetch_cube(F4) instead.

Reply:
This was not supposed to be pushed. It slipped though on commit